qutebrowser/keyinput/modeman.py

```diff
@@ -310,6 +312,24 @@ def eventFilter(self, event):
         Return:
             True if event should be filtered, False otherwise.
         """
+
+        # The keypress to start "show key mode" also ends up here
+        if self._show_key == 1:
+            self._show_key = 2
```

Comment:
I think it'd be nicer to make this an enum, e.g. `ShowKeyState = usertypes.enum('ShowKeyState', 'ignore, capture')` on class level, and then something like this here:

```python
if self._show_key == self.ShowKeyState.ignore:
    self._show_key = self.ShowKeyState.capture

...
```